samples/snippets/src/main/java/com/example/spanner/AsyncQueryExample.java

```diff
@@ -71,18 +71,18 @@ public CallbackResponse cursorReady(AsyncResultSet resultSet) {
                   try {
                     while (true) {
                       switch (resultSet.tryNext()) {
-                        // OK: There is a row ready.
+                          // OK: There is a row ready.
```

Comment:
@Neenu1995 it seems like owlbot is wrongly formatting these comments (whenever it re-runs). We don't have any special checkstyle configuration, so even though we can manually fix this, owlbot will break it whenever it re-runs.

Is there anything we can do in owlbot to fix this?